medical-portal/src/UI/src/app/case-details/case-details.component.ts

```diff
@@ -165,6 +171,9 @@ export class CaseDetailsComponent implements OnInit {
               doc.submittalStatus as SubmittalStatusEnum,
             )
           ) {
+            if (doc.documentType == DocumentTypeEnum.DMER) {
```

Comment:
I assume in a future ticket, this will also check that you have claimed it or someone in your network has claimed it?